packages/wonder-blocks-core/util/types.js

```diff
@@ -210,6 +210,8 @@ export type TextViewSharedProps = {
 
     id?: string,
 
+    tag?: string,
```

Comment:
Good call.  There's definitely some tags that don't make sense.  This means we won't be able to share the type for `tag` between `View` and `Text` since different tags make sense for these components.